h/authService.h

```diff
@@ -26,7 +26,17 @@
 #include "httpserver.h"
 #include "dataservice.h"
 
+#define PASSWORD_RESET_OK                0
```

Comment:
Are you going to use these outside the C file? If not, please move them and add some prefix to indicate whether this is an RC, RSN or something else.